### ISSUE TITLE
Dexter.checkPermissionOnSameThread crashes when Looper has already been prepared

### ISSUE THREAD

**Issue by vincentjames501** [Karumi/Dexter#50]
### Expected behaviour
Calling ```Dexter.checkPermissionOnSameThread``` shouldn't crash if a looper has already been prepared

### Actual behaviour
```
03-14 17:00:16.549 16003-16614/com.blah E/AndroidRuntime: FATAL EXCEPTION: IntentService[GeofenceTransitionsIntentService]
                                                                       Process: com.blah, PID: 16003
                                                                       java.lang.RuntimeException: Only one Looper may be created per thread
                                                                           at android.os.Looper.prepare(Looper.java:82)
                                                                           at android.os.Looper.prepare(Looper.java:77)
                                                                           at com.karumi.dexter.WorkerThread.<init>(WorkerThread.java:30)
                                                                           at com.karumi.dexter.ThreadFactory.makeSameThread(ThreadFactory.java:40)
                                                                           at com.karumi.dexter.Dexter.checkPermissionOnSameThread(Dexter.java:60)
                                                                           at ...
```

### Steps to reproduce
In a background thread, call ```Looper.prepare()``` before calling ```Dexter.checkPermissionOnSameThread```

### Version of the library
2.2.1

### Fix
According to stack overflow adding the following to ```WorkerThread.java``` should fix it (or at least this exception):

```java
if (Looper.myLooper() == null) {
    Looper.prepare();
}
```

I haven't tested the fix but it should work. I'll see about doing a pull request soon but someone feel free to beat me to it.

**Comment by Serchinastico** [comment]
Thanks for reporting @vincentjames501 !

We'll wait for your (or others) PR and merge it ASAP

**Comment by Serchinastico** [comment]
We just released version 2.2.2 with your code.

Thanks again for your report & fix!
